src/main/java/com/mitchej123/hodgepodge/client/HodgepodgeClient.java

```diff
@@ -67,6 +68,11 @@ public static void postInit() {
             meta.description = "This IC2 Addon allows you to simulate the CropBreeding.";
             meta.autogenerated = false;
         }
+
+        if (Compat.isBiomesOPlentyPresent()) {
+            // removes the popup that BOP shows on first world gen
+            MinecraftForge.EVENT_BUS.unregister(WorldTypeMessageEventHandler.instance);
```

Comment:
I tested it in the HP dev env with BOP set to compileonly and had no issues

Reply:
I think the way class loading works is that it will try to load the class when it executes that line of code, so no problem if you keep it inside a Loader.isModLoaded